packages/ckeditor-cloud-services-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ckeditor/ckeditor-cloud-services-core",
-  "version": "22.0.0",
+  "version": "22.0.1",
```

Comment:
This change should be reverted. Our release scripts bump versions automatically.

Reply:
Reverted